test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMeanShort.test.cpp

```diff
@@ -0,0 +1,224 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Types/Index.hpp>
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+
+#include <OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Duration.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Derived.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Derived/Angle.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Length.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMeanShort.hpp>
+
+#include <Global.test.hpp>
+
+using ostk::core::types::Real;
+using ostk::core::types::Index;
+using ostk::core::ctnr::Tuple;
+
+using ostk::math::obj::Vector6d;
+using ostk::math::obj::Vector3d;
+
+using ostk::physics::coord::Position;
+using ostk::physics::coord::Velocity;
+using ostk::physics::coord::Frame;
+using EarthGravitationalModel = ostk::physics::environment::gravitational::Earth;
+using ostk::physics::time::Duration;
+using ostk::physics::units::Angle;
+using ostk::physics::units::Derived;
+using ostk::physics::units::Length;
+
+using ostk::astro::trajectory::orbit::models::kepler::BrouwerLyddaneMeanShort;
+using ostk::astro::trajectory::orbit::models::kepler::COE;
+
+TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Kepler_BrouwerLyddaneMeanShort, Constructor)
+{
+    {
+        const Length semiMajorAxis = Length::Kilometers(7000.0);
+        const Real eccentricity = 0.1;
+        const Angle inclination = Angle::Degrees(10.0);
+        const Angle raan = Angle::Degrees(20.0);
+        const Angle aop = Angle::Degrees(30.0);
+        const Angle meanAnomaly = Angle::Degrees(40.0);
+
+        EXPECT_NO_THROW(BrouwerLyddaneMeanShort(semiMajorAxis, eccentricity, inclination, raan, aop, meanAnomaly););
+    }
+}
+
+class OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Kepler_BrouwerLyddaneMeanShort_Parametrized
+    : public ::testing::TestWithParam<std::tuple<Position, Velocity, Vector6d>>
+{
+};
+
+INSTANTIATE_TEST_SUITE_P(
+    Cartesian,
+    OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Kepler_BrouwerLyddaneMeanShort_Parametrized,
+    ::testing::Values(std::make_tuple(
+        Position::Meters({6596065.624114551, 2282234.953292401, -18030.93992064121}, Frame::GCRF()),
+        Velocity::MetersPerSecond({345.4716519563907, -967.0404288726759, 7488.686029827369}, Frame::GCRF()),
+        Vector6d {
+            6973743.736075629,
+            0.001202049190851806,
+            1.7071571281899047,
+            0.3327524364104959,
+            1.568634993642334,
+            4.7143446820819745,
+        }
+    ))
+);
```

Comment:
Guessing these are canonical examples? Or are these values obtained when running OSTk with the impl?

Reply:
These are values from GMAT. That's a good point I'll put that in a comment.